### ISSUE TITLE
Test runtime errors after recent change of Wiremock packaging used

### ISSUE THREAD

**Issue by apankowski** [kotest/kotest-extensions-wiremock#39]
While trying to upgrade the extension in one of my projects I encountered suspicious test failures, looking more or less like this:

```
java.lang.reflect.InvocationTargetException
  Caused by: java.lang.IncompatibleClassChangeError: class org.eclipse.jetty.http2.server.HttpChannelOverHTTP2 has interface org.eclipse.jetty.server.HttpChannel as super class
```

Tracking it down, I saw that in https://github.com/kotest/kotest-extensions-wiremock/pull/36 we have upgraded WireMock to 3.3.1, which is great.

However, it has not _just_ changed coordinates of the WireMock package.

We've also switched from [standalone WireMock variant](https://wiremock.org/docs/standalone/) having all its dependencies shaded and bundled within the main JAR (POM: [org.wiremock:wiremock-standalone:3.3.1](https://search.maven.org/artifact/org.wiremock/wiremock-standalone/3.3.1/jar)) to a regular one having its dependencies declared in POM and pulled in (POM: [org.wiremock:wiremock:3.3.1](https://search.maven.org/artifact/org.wiremock/wiremock/3.3.1/jar)).

Problem is, using the regular version may change versions of those dependencies of the project that are also declared by WireMock. As an example, if my project depends (even transitively) on `org.apache.httpcomponents.client5:httpclient5:4.5.6` and I use (even transitively) `org.wiremock:wiremock:3.3.1`, the dependency will get upgraded to `org.apache.httpcomponents.client5:httpclient5:5.2.1` (since that's the version declared by WireMock, and it's higher), which can lead to errors, as in my case.

I think this is exactly why standalone variant of WireMock exists and why Wiremock Kotest extension used it (correctly) to being with.

### TL;DR

To minimize impact on the projects and avoid surprises, I think we should stick to standalone WireMock variant:

```toml
wiremock = { module = "org.wiremock:wiremock-standalone", version.ref = "wiremock" }
```

### Workaround

For anyone else living on the edge, the following workaround did the trick for me:

```kotlin
dependencies {
  testImplementation("org.wiremock:wiremock-standalone:3.3.1")
  testImplementation("io.kotest.extensions:kotest-extensions-wiremock") {
    // Exclude WireMock packaging variant that interferes with project's dependency versions
    // Remove if / when the extension switches to org.wiremock:wiremock-standalone
    exclude("org.wiremock", "wiremock")
  }
}
```

**Comment by Kantis** [comment]
Thanks for raising the issue. I think it wasn't intended to switch from the standalone version
